server/controllers/donations.js

```diff
@@ -46,7 +46,7 @@ const stripeDonation = (req, res, next) => {
       interval
     }
   })
-  .then(() => res.send({success: true, user: req.user.info }))
+  .then((transaction) => res.send({success: true, user: req.user.info, transaction }))
```

Comment:
Can you put a comment above this explaining why we are returning `transaction` and a TODO that it needs to be reworked when we split the flow? Or add a test that will break if we split the flow.

Longer term, since this will be async, an email with redirect link (that confirms the donation) would be better.